src/PublicApiGeneratorTests/PublicApiGeneratorTests.csproj

```diff
@@ -9,6 +9,9 @@
     IDE0060 Remove unused parameter 'name' if it is not part of a shipped public API
     IDE1006 Naming rule violation: These words must begin with upper case characters
     -->
+    <NoWarn>$(NoWarn);CS0067</NoWarn>
```

Comment:
Merge this with NoWarns above.

Reply:
I'm confused. I'm 100% positive that I did not type this. Maybe I clicked something in VS by mistake.

Reply:
Could have been git automerging when I rebased.